build.gradle.kts

```diff
@@ -0,0 +1,55 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath("org.sonarsource.scanner.gradle:sonarqube-gradle-plugin:2.7")
```

Comment:
Why are these plugin dependencies duplicated? They already exist in the `plugins` block.

Reply:
Do they not need to be imported as buildscript dependencies before applying them in the `plugins` block?

Reply:
Nope, though I see where the confusion might come from. In Gradle 4, you had to add plugins as dependencies to the buildscript classpath:

```
buildscript {
  dependencies {
    classpath("xyz")
  }
}

apply plugin: "xyz"
```

Since Gradle 5, using the `plugins { ... }` syntax handles both for you:
```
plugins {
  id("xyz")
}
```